patches/arcade/0009-Add-reference-to-PlatformAbstractions.patch

```diff
@@ -1,15 +1,15 @@
-From db4c528bb2335f791f46f76f62f392597e298ef0 Mon Sep 17 00:00:00 2001
-From: dseefeld <dseefeld@microsoft.com>
-Date: Wed, 1 Apr 2020 16:32:18 +0000
-Subject: [PATCH] Add reference to PlatformAbstractions
+From db4c528bb2335f791f46f76f62f392597e298ef0 Mon Sep 17 00:00:00 2001  
+From: dseefeld <dseefeld@microsoft.com> 
+Date: Wed, 1 Apr 2020 16:32:18 +0000    
+Subject: [PATCH] Add reference to PlatformAbstractions  
 
-Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk.csproj doesn't have a reference
-to PlatformAbstractions. It builds in the production build because it has a
-reference to Microsoft.Extensions.DependencyModel, which also brought in
-PlatformAbstractions. That dependency has been removed with
-https://github.com/dotnet/core-setup/pull/5218.
-The tarball build uses a newer version of Microsoft.Extensions.DependencyModel
-with the dependency removed, so an explicit dependency needs to be added now until
+Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk.csproj doesn't have a reference    
+to PlatformAbstractions. It builds in the production build because it has a 
+reference to Microsoft.Extensions.DependencyModel, which also brought in    
+PlatformAbstractions. That dependency has been removed with 
+https://github.com/dotnet/core-setup/pull/5218. 
+The tarball build uses a newer version of Microsoft.Extensions.DependencyModel  
+with the dependency removed, so an explicit dependency needs to be added now until  
```

Comment:
Weird patch changes, solely adding trailing whitespace... rebase gone bad?

Reply:
That is weird.  I'll fix it.